.github/workflows/pr-vrt.yml

```diff
@@ -12,7 +8,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  NX_PARALLEL: 4 # ubuntu-latest = 4-core CPU / 16 GB of RAM | macos-14-xlarge (arm) = 6-core CPU / 14 GB of RAM
+  NX_PARALLEL: 6 # ubuntu-latest = 4-core CPU / 16 GB of RAM | macos-14-xlarge (arm) = 6-core CPU / 14 GB of RAM
```

Comment:
we can use powerful ARM agents ( generation is executed via webpack and storywright ( playwright ) )